test/template/node-javascript.mustache

```diff
@@ -62,8 +62,8 @@ function run_javascript_tests(test_obj, Urlencoded, js_beautify, html_beautify,
         // }
 
         if (opts.indent_size === 4 && input) {
-            wrapped_input = '{\n' + input.replace(/^(.+)$/mg, '    $1') + '\n    foo = bar;\n}';
-            wrapped_expectation = '{\n' + expectation.replace(/^(.+)$/mg, '    $1') + '\n    foo = bar;\n}';
+            wrapped_input = '{\n' + input.trim().replace(/^(.+)$/mg, '    $1') + '\n    foo = bar;\n}';
```

Comment:
You really shouldn't put trim here...  It means your tests aren't testing what they intend to.